GameData/RealismOverhaul/Engine_Configs/E1_Config.cfg

```diff
@@ -79,8 +80,8 @@
 		CONFIG
 		{
 			name = E-1
-			minThrust = 1884.59 // 380klbf sea level
-			maxThrust = 1884.59
+			minThrust = 1984.58 // 380klbf sea level - [A] 400klbf instead of 380
```

Comment:
That certainly might be the case. Sutton does not make an obvious distinction between theoretical and practical, but I had assumed practical due to the context of one of the figures provided. Given that these communications were provided later than that datasheet was written/unclassified, it could be a time difference... However, I do not know how long the production for a 400klbf E-1 continued, so that might be irrelevant.
![Screenshot_0801_171752](https://user-images.githubusercontent.com/38171736/127789839-06444cc5-424c-45f0-8746-3ec62a4b484e.png)

Reply:
Uh, there _was_ no production AFAIK. The desired goal thrust was 400klbf the but the engine was canceled before it entered production (or even mass prototype) level. My guess is that what's going on here is that it is described as a "400klbf-class" engine in internal documents, in the same way that F-1 was a "1mllbf" engine which later became 1.5mlbf and then 1.522mlbf.